pages.fa/common/hexo.md

```diff
@@ -0,0 +1,28 @@
+# hexo
+
+> چارچوب وبلاگ سریع، ساده وقدرتمند.
+> اطلاعات بیشتر: <https://hexo.io/>.
+
+- ساخت اولیه یک وبسایت:
+
+`hexo init {{path/to/directory}}`
+
+- ایجاد یک مقاله جدید:
+
+`hexo new {{layout}} {{title}}`
+
+- تولید فایل های ایستا:
+
+`hexo generate`
+
+- شروع یک سرور محلی:
+
+`hexo server`
+
+- پیاده سازی وبسایت:
+
+`hexo deploy`
+
+- پاک کردن فایلها کش و تولید فایلهای پابلیک:
```

Comment:
Here, on the English page, additional files/directories are present too; if possible can you add the same here?

`  Clean the cache file (db.json) and generated files (public/):`

Reply:
Nice catch, thank you.